.eslintrc

```diff
@@ -32,6 +32,23 @@
         "array": false,
         "object": true
       }
+    ],
+    "curly": ["error", "all"],
+    "newline-before-return": "error",
+    "sort-class-members/sort-class-members": [
+      2,
+      {
+        "order": [
+          "[static-properties]",
+          "[static-methods]",
+          "[properties]",
+          "[conventional-private-properties]",
+          "constructor",
+          "[methods]",
+          "[conventional-private-methods]"
+        ],
+        "accessorPairPositioning": "getThenSet"
+      }
     ]
```

Comment:
@jshjohnson `sort-class-members` is lovely ESLint plugin with one major drawback - no built-in autofix.
So, for pleasure of lazy developers (like me), I would suggest to do

```sh
npm i -D jscodeshift sort-class-members-codemod
```

and edit `lint-staged.config.js`:

```js
module.exports = {
  "*.js": [
    "jscodeshift -t node_modules/sort-class-members-codemod/index",
    "eslint --fix --quiet -f visualstudio",
    "git add"
  ]
  // rest of file here
};
```

Reply:
Hmmm i'm sure it was auto-fixing for me 🤔

Reply:
Nope, unfortunately. But lint-staged can help.

Reply:
Ooops, my bad! They had a new version with autofix! :tada: